cross-modal-search/app.py

```diff
@@ -23,8 +23,7 @@ def config():
 
 
 def index_restful():
-    flow = Flow().load_config('flows/flow-index.yml')
-    flow.use_rest_gateway()
+    flow = Flow().load_config('flows/flow-index.yml', override_with={'protocol': 'http'})
```

Comment:
this I am not sure how is it working? The `protocol` should be in the `flow-index.yml` as an ENV var

Reply:
As discussed, a test is added to core to test`Flow.load_config` with `override_with` [here](https://github.com/jina-ai/jina/pull/2901).